src/components/ChooseDialect.js

```diff
@@ -33,7 +33,10 @@ const ChooseDialect = ({ navigation }) => {
           />
         );
       })}
-      <Button mode="contained" onPress={() => navigation.navigate("Lesson")}>
+      <Button
+        mode="contained"
+        onPress={() => navigation.navigate("PatternList")}
```

Comment:
Please keep this as `navigation.navigate("Lesson")`

Reply:
Change button text in ChooseDialect to 'select pattern'

Reply:
Fixed.